### ISSUE TITLE
Should not intercept Ctrl+Alt+Click

### ISSUE THREAD

**Issue by LukaszMendakiewicz** [Microsoft/VS-PPT#198]
Ctrl+Alt+Click may be used by the IDE to perform other operations, but this extension will still intercept this combo. It should only react to Ctrl+Click.

**Comment by kapsiR** [comment]
Yes, especially it breaks the new Multi-Caret Support in VS 15.8 (AddCaret: Ctrl + Alt + Click)
